install.sh

```diff
@@ -53,6 +53,19 @@ EOF
   shift $((${OPTIND} - 1))
   UNKNOWN_ARGS="$*"
 
+# Check for prerequisite tooling
+# https://atlassian-labs.github.io/data-center-terraform/userguide/PREREQUISITES/
+check_for_prerequisites() {
+  declare -a tools=("aws" "helm" "terraform" "jq")
```

Comment:
```suggestion
  declare -a tools=("aws" "helm" "terraform")
```

`jq` is not necessary